templates/cisco_nxos_show_environment.textfsm

```diff
@@ -0,0 +1,54 @@
+Value Required POWER_SUPPLY (\d+)
```

Comment:
We should re-use the other template and then add the additional fixes required for #790.

Also, make sure the values are the same from the old template as well.